server/auth/auth.js

```diff
@@ -12,7 +12,7 @@ export class Helper {
    * @param {string} email - users email
    */
 	static isValidEmail(email) {  /**hint from code mentor tutorials @OlawaleAladeusi*/
-		return /\S+@\S+\.\S+/.test(email);
+		return (/\S+@\S+\.\S+/.test(email) && /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/.test(email));
```

Comment:
#### Critical
Please remove that comment. Make sure you understand this construct before implementing it. It's better to get understanding on this rather than state the source without being able to explain it.